app/views/Project/Tagging/Sources/components/EditableEntry/index.tsx

```diff
@@ -248,6 +285,31 @@ function EditableEntry(props: Props) {
         </Button>
     );
 
+    const handleEntryDeleteButtonClick = useCallback(() => {
+        deleteEntry({
+            variables: {
+                projectId,
+                entryId: entry.id,
+            },
+        });
+    }, [projectId, entry.id]);
+
+    const entryDeleteButton = (
+        <ConfirmButton
+            name={undefined}
+            variant="secondary"
+            onClick={handleEntryDeleteButtonClick}
+            message="Are you sure you want to delete the entry?"
+            disabled={deleteEntryPending}
```

Comment:
Should also be disabled on edit mode.
Let's disable all other actions on edit mode